src/colorizer/shaders/colorize_RGBA8U.frag

```diff
@@ -9,7 +9,15 @@ uniform float featureMax;
 uniform vec2 canvasToFrameScale;
 uniform sampler2D colorRamp;
 uniform vec3 backgroundColor;
+
+const uint DRAW_MODE_HIDE = 0u;
```

Comment:
it might be useful to have a comment above these to say they must be kept in sync with the enum in ColorizeCanvas (and vice versa)